src/main/java/it/eng/idsa/businesslogic/service/impl/ClearingHouseServiceImpl.java

```diff
@@ -100,7 +100,7 @@ public String createProcessIdAtClearingHouse(String senderToken, String contract
 						response.code(), response.message(), response.body().string());
 				return null;
 			} else {
-				logger.info("Clearing House created a new log ProcessID: {}", contractAgreementUUID);
+				logger.info("Clearing House created a new log ProcessID");
```

Comment:
This is not considered as sensitive data, we could leave that in log - contractAgreementUUID

Reply:
Same for this one